protocol/contracts/interfaces/basin/pumps/ICappedReservesPump.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+pragma experimental ABIEncoderV2;
+
+/**
+ * @title ICappedReservesPump
+ * @notice Provides an interface for Pumps which capped
+ * reserves through the use of a cumulative reserve.
+ */
+interface ICappedReservesPump {
+    // TODO: Maybe not the best place to put this, perhaps in a MultiFlowPump interface?
```

Comment:
This is fine imo

Reply:
Removed comment here: https://github.com/BeanstalkFarms/Beanstalk/commit/bc04fee61d1f0cbb0f6935a342ea3bf22e4793f1